rust/xaynet-core/src/message/mod.rs

```diff
@@ -1,12 +1,5 @@
 //! The messages of the PET protocol.
 //!
-//! The [`Message`]s adhere to the common [`Header`] and [`Payload`] scheme. They are parsed via
-//! [`MessageBuffer`]s which provide a checked access to the raw message, whereas the serialization
-//! and deserialization is implemented via the [`ToBytes`] and [`FromBytes`] traits.
-//!
-//! All messages are signed and encrypted in an authenticated encryption scheme via [`MessageSeal`]
-//! and [`MessageOpen`], which employ the wrappers of the [crypto module].
-//!
```

Comment:
did you intend to remove this?

Reply:
Yeah. There's no more `Header`, `MessageSeal` and `MessageOpen` types, and the `ToBytes` and `FromBytes` traits are not actually being used for `Message`, so the section is pretty outdated.